index.html

```diff
@@ -0,0 +1,191 @@
+                    <div class="profile-description">
+                        <h4>CANOPY</h4>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>Back End Dev</p>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>2015</p>
+                    </div>
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="profile"><img src="image/Snapshoot Portfolio (2).png" alt="Snapshoot"></div>
+                <div class="card-description">
+                    <h3>Tonic</h3>
+                    <div class="profile-description">
+                        <h4>CANOPY</h4>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>Back End Dev</p>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>2015</p>
+                    </div>
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="profile"><img src="image/Snapshoot Portfolio (3).png" alt="Snapshoot"></div>
+                <div class="card-description">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <div class="profile-description">
+                        <h4>CANOPY</h4>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>Back End Dev</p>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>2015</p>
+                    </div>
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
```

Comment:
- [ ] Kindly make sure you have the correct fonts (family and size on your element)
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212125774-8a7cc254-7947-4e80-8ef9-691d2f74c2b4.png)
Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212125907-069af0f8-2733-4256-865a-bb257f19ea05.png)